app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
```

Comment:
Storing `self` in a class attribute `people` can lead to unexpected behavior if the class is used in different contexts. Consider using a different approach to manage `Person` instances.